### ISSUE TITLE
Biospec Dshbrd - Locking the collection so changes can't be made

### ISSUE THREAD

**Issue by wyattks** [episphere/connect#216]
We would like the Specimen Collection to be "locked" so changes can no longer be made to the collection screen.  Each individual tube should be "Locked" and changes not allowed when they are shipped to NCI processing center.  The collection as a whole needs to be locked 3 days after the first tubes are shipped.  This leaves it open for straggler tubes but doesn't leave it open indefinitely.

**Comment by wyattks** [comment]
Closing - will open under new issue if needed once site feedback is complete.
